### DISCUSSION TITLE
How to import the `var foo` from a `.js` file, which doesn't have `export`?

### DISCUSSION THREAD

**Discussion by nyngwang** [webpack/webpack#15732] [Q&A] [answered]
As title. The `.js`-file is machine-generated, so I should not edit it directly. But now I have to import these globals. Do we have any plugin to achieve this? So I can write:

```
import './path/to/the/generated/foobar.js'
```

and then the `var foo` can be accessed from the other modules as if it's a global variable.

**Comment by vankop** [comment, marked as answer]
you can use loader to modify js file

**Reply by nyngwang** [reply]
Which one?

**Reply by vankop** [reply]
you can create a loader to modify this module before building
